airgun/views/common.py

```diff
@@ -445,6 +446,48 @@ def search(self, query):
         return self.table.read()
 
 
+class SearchableViewMixinPF4(SearchableViewMixin):
+    """Mixin which adds :class:`airgun.widgets.Search` widget and
+    :meth:`airgun.widgets.Search.search` to your view. It's useful for _most_ entities list views
+
+    where searchbox and results table are present.
+    Note that class which uses this mixin should have :attr: `table` attribute.
+    """
+
+    searchbox = PF4Search()
+    blank_page = Text("//div[contains(@class, 'pf-c-empty-state')]")
+
+    def is_searchable(self):
+        """Verify that search procedure can be executed against specific page
+        that is not blank
+        """
+        if self.searchbox.search_field.is_displayed and (not self.blank_page.is_displayed):
```

Comment:
Since you're trying to return the boolean, you can just return the statement directly.

Non blocking, but something to keep in mind for things like this.

```
return self.searchbox.search_field.is_displayed and not self.blank_page.is_displayed
```